pkg/controller/core/clusterqueue_controller.go

```diff
@@ -56,10 +56,6 @@ import (
 
 const snapshotWorkers = 5
 
-type ClusterQueueUpdateWatcher interface {
```

Comment:
I don't understand why this is changed. Can you instead call the already existing update on topologyUpdateWatchers?

Reply:
I can't subscribe for `ClusterQueueUpdateWatcher` due to it is on the core `SetupControllers`.

Reply:
No, we need to wait for update of clusterQueue.

Reply:
I need to know about this changes

https://github.com/kubernetes-sigs/kueue/blob/bf8b06861d999fe892e4c0dcfa3c26716af28610/pkg/controller/tas/topology_controller.go#L179-L180

Reply:
Again, this problem is already solved, please look into the previous referenced PR and I expect you will be able to follow the approach.

Reply:
Which PR do you mean?

Reply:
https://github.com/kubernetes-sigs/kueue/pull/3770

Reply:
Done